actionpack/lib/action_controller/renderer.rb

```diff
@@ -71,6 +71,18 @@ def initialize(controller, env, defaults)
     end
 
     # Render templates with any options from ActionController::Base#render_to_string.
+    #
+    # The primary options are:
+    # * <tt>:partial</tt> - See <tt>ActionView::PartialRenderer</tt> for details.
+    # * <tt>:file</tt> - Renders an explicit template file. Add <tt>:locals</tt> to pass in, if so desired.
+    # * <tt>:inline</tt> - Renders an inline template.
```

Comment:
I would mention that this is a template string.

Reply:
How about?  `Renders an ERB template string.`

Reply:
I like that change